### DISCUSSION TITLE
Challenge 5 Issues ** Spoiler **

### DISCUSSION THREAD

**Discussion by RichardPalladino** [Cyfrin/foundry-full-course-cu#149] [Q&A] [answered]
I'm sure that I've been submitting the correct value for Challenge 5 password because I literally copy the input data value from past transactions that resulted in successful solution.  Still, I'm receiving the message from Metamask as if somehow there is an error.

I don't see any evidence for password having changed.  Is there a reason copying the decoded input data (which is the same for every past transaction I've checked) for the solution somehow wouldn't work?  

I was thinking possibly there was a special character in there but I saw no evidence for that.  Any hint on what I might be missing?

**Comment by RichardPalladino** [comment]
I'm going to add that I get it to work correctly on Sepolia, but Arbitrum gives me errors.  (And I believe the solution would be the same since the code is the same.)  I even tried writing an interface in Remix to call the solveFunction() from something other than arbiscan interface and it still didn't work.

**Comment by RichardPalladino** [comment, marked as answer]
I figured it out, and feel like a major doofus.

It turns out _I had already submitted correctly to the challenge and minted an NFT_.  I guess there are controls in place to make sure each NFT can only be mined once per wallet address.

It was late.  I obviously wasn't paying much attention.  Whoops.

**Reply by cromewar** [reply]
Don't worry, and thanks for explaining how you solved this.
